apps/web-giddh/src/app/home/components/bank-accounts/bank-accounts.component.ts

```diff
@@ -50,6 +53,12 @@ export class BankAccountsComponent implements OnInit, OnDestroy {
     public isGocardlessSupportedCountry: boolean;
     /** True, if is integration module are in scope  */
     public hasIntegrationScope: boolean = false;
+    /** True, if is bank is connected */
+    public isBankisConnected: boolean;
```

Comment:
```suggestion
    public isBankConnected: boolean;
```
make changes in overall file where it is use

Reply:
done and removed this varible because this varible is not used anywhere